lib/routes/yomiuri/news.js

```diff
@@ -64,7 +122,7 @@ module.exports = async (ctx) => {
     );
 
     ctx.state.data = {
-        title: '読売新聞-' + category_name,
+        title: '読売新聞 - ' + category_name,
```

Comment:
If category name is empty. Don't append the '-'

![Screenshot_2021_0712_095249.jpg](https://user-images.githubusercontent.com/19922286/125219234-10b48e80-e2f7-11eb-88c9-8bcb190edba8.jpg)

Reply:
Hello, @